49-hvac-current-monitor/firmware/src/main.cpp

```diff
@@ -0,0 +1,330 @@
+#include <Notecard.h>
+#include "NotecardEnvVarManager.h"
+
+// Uncomment this line and replace com.my-company.my-name:my-project with your
+// ProductUID.
+// #define PRODUCT_UID "com.my-company.my-name:my-project"
+
+#ifndef PRODUCT_UID
+#define PRODUCT_UID ""
+#pragma message "PRODUCT_UID is not defined in this example. Please ensure your Notecard has a product identifier set before running this example or define it in code here. More details at https://bit.ly/product-uid"
+#endif
+
+#ifndef OUTBOUND_MINS
+#define OUTBOUND_MINS 10
+#endif
+
+#ifndef INBOUND_MINS
+#define INBOUND_MINS 5
+#endif
+
+#ifndef MAX_SERIAL_WAIT_MS
+#define MAX_SERIAL_WAIT_MS 5000
+#endif
+
+#ifndef ENV_FETCH_INTERVAL_MS
+#define ENV_FETCH_INTERVAL_MS (3 * 60 * 1000) // 3 minutes
+#endif
+
+#ifndef ANALOG_PIN
+#define ANALOG_PIN A0
+#endif
+
+#ifndef ADC_BITS
+#define ADC_BITS 10
+#endif
+
+#define ADC_RANGE (1 << ADC_BITS)
+#define DEFAULT_ADC_OFFSET (ADC_RANGE >> 1)
```

Comment:
You might want to comment this to make it obvious for anyone not familiar with bit shifting, and the half full-range offset.

Reply:
I just realized this isn't used anymore, so I'll remove it altogether.